bin/run-keystone-build.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * This file is based on @keystonejs/keystone/bin/commands/build.js
+ * The main reason is to better memory control!
+ */
+
+const path = require('path')
+
+const chalk = require('chalk')
+
+// NOTE: force to production (form keystone build.js)
+// NOTE: it's required to set before the conf initialization
+process.env.NODE_ENV = 'production'
+process.env.PHASE = 'build'
+
+const { prepareKeystoneInstance } = require('@open-condo/keystone/prepareKeystoneApp')
+
+const CWD = process.cwd()
+
+async function main () {
+    const index = path.resolve('./index.js')
+
+    console.log(chalk.gray.bold(`KEYSTONE 🚀 prepare keystone instance. index=${index}`))
+    const { keystone, apps, distDir } = await prepareKeystoneInstance(index, {
+        excludeApps: [
+            'HealthCheck', 'RequestCache', 'AdapterCache', 'VersioningMiddleware', 'OIDCMiddleware',
```

Comment:
We should not hard-code this ones, this means each new middleware should be added here.

That's bad. You should handle it gracefully

Reply:
It's just got speed up. Really you can skip new. It's just a small speed up.
And you really don't need to add all or new